db/migrate/20200428102014_add_bulk_invite_link_to_invites.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddBulkInviteLinkToInvites < ActiveRecord::Migration[6.0]
+  def change
+    add_column :invites, :max_redemptions_allowed, :integer, null: false, default: 1
+    add_column :invites, :redemption_count, :integer, null: false, default: 0
+    add_column :invites, :expires_at, :datetime, null: true
+
+    execute <<~SQL
+      UPDATE invites SET expires_at = updated_at + INTERVAL '#{SiteSetting.invite_expiry_days} days'
```

Comment:
We need to hardcode `SiteSetting.invite_expiry_days` here otherwise this migration will break when the site setting is removed in the futuer.

Reply:
We can hardcode the default value of `30`, but what about sites that have modified the site setting?

Reply:
O this isn't a new site setting. We can write a SQL query to fetch the value then.